pkg/runctl/controller.go

```diff
@@ -432,14 +432,17 @@ func (c *Controller) updateStateAndResult(pipelineRun k8s.PipelineRun, state api
 
 func (c *Controller) commitStatusAndMeter(pipelineRun k8s.PipelineRun) error {
 	start := time.Now()
-	finishedStates, err := pipelineRun.CommitStatus()
+	finishedStates, retryDone, err := pipelineRun.CommitStatus()
 	if err != nil {
 		klog.V(6).Infof("commitStatus failed with error %s", err.Error())
 		return err
 	}
 	end := time.Now()
 	elapsed := end.Sub(start)
 	klog.V(6).Infof("commit of %q took %v", pipelineRun.String(), elapsed)
+	if retryDone {
+		c.metrics.ObserveRetryDurationByType("UpdateState", elapsed)
+	}
 	c.metrics.ObserveUpdateDurationByType("UpdateState", elapsed)
```

Comment:
Just clarification: it is intended that in case of a retry we call `ObserveRetryDurationByType` and ObserveUpdateDurationByType` ?